nextflow.config

```diff
@@ -181,6 +181,11 @@ params {
     gprofiler2_min_diff                     = 1
     gprofiler2_palette_name                 = 'Blues'
 
+    // DECOUPLER options
+    decoupler_run                   = false
```

Comment:
I think it should become an option for the `functional_method` parameter instead.